woocommerce/woocommerce-android
# Issue/2783 variation sale

Fixes #2783.

**To test:**

1. Go to the Products tab.
2. Open a variable product.
3. Tap on Variations to open the variations list.
4. Tap on a variation to open the variation details.
5. Tap on Price.
6. Enter a sale price for that variation.
7. Tap "Done" to return to the variation detail.
8. Tap "Update" to save your changes.
9. Go back to the variations list and notice that it shows the sale price for that variation
